index.html

```diff
@@ -2999,32 +2999,56 @@ <h3>Trust Model</h3>
         <ul>
           <li>
 The [=verifier=] trusts the [=issuer=] to issue the [=credential=] that
-it received. To establish this trust, a [=credential=] is expected to either:
+it received after verification has been performed. 
+To establish this trust, a [=credential=] is expected to either:
             <ul>
               <li>
-Secure the [=credential=] with a <a href="#securing-mechanisms">securing mechanism</a> establishing that the
+Be secured with a <a href="#securing-mechanisms">securing mechanism</a> establishing that the
 [=issuer=] generated the [=credential=] (that is, it is a
 [=verifiable credential=]), or
               </li>
               <li>
 Have been transmitted in a way clearly establishing that the [=issuer=]
-generated the [=verifiable credential=] and that the
-[=verifiable credential=] was not tampered with in transit or storage. This
+generated the [=credential=] and that the
+[=credential=] was not tampered with in transit or storage. This
 trust could be weakened depending on the risk assessment of the [=verifier=].
               </li>
             </ul>
           </li>
           <li>
 All [=entities=] trust the [=verifiable data registry=] to be
 tamper-evident and to be a correct record of which data is controlled by which
-[=entities=].
+[=entities=]. This is typically achieved by the method of its publication. 
+This could be by a peer to peer protocol from a trusted publisher, or 
+on a publicly accessible and well known web site (with a content hash), 
+or on a blockchain etc. When entities publish meta data about themselves
+they can be integrity protected by signing with the entity's private key.
           </li>
           <li>
 The [=holder=] and [=verifier=] trust the [=issuer=] to issue
 true (that is, not false) [=credentials=] about the [=subject=], and to
```

Comment:
Truth (or falsity) is not part of our remit. *It is entirely possible that a holder and/or verifier might trust that everything issued by a given issuer is false*. All that a verifier can trust is that the issuer made the claims contained in a properly secured verifiable credential.

Reply:
Possibly this should be a separate PR since it is the current CR wording.

Reply:
This PR says it is to "[Update Trust Section](https://github.com/w3c/vc-data-model/pull/1469#top)", of which this paragraph is a part. Why is it out of bounds for this PR?

Reply:
This PR is specifically to address the issues raised by Jeffrey. I did not want to update other parts of the trust section that were not impacted by these issues. In other words, the changes you are suggesting are additional to Jeffey's comments, so they should be raised in a separate PR that addresses what you believe are bugs in the current text, so that even if the current PR is rejected, your proposed changes can be judged independently on their own merit in a separate PR. I did not want to muddy the waters by combining the two.

Reply:
> This PR is specifically to address the issues raised by Jeffrey 

Please adjust this PR's title to indicate such, e.g.,

> address issues raised by Jeffrey Yasskin on section 5.2 (Trust Model)

Reply:
Done